### PR TITLE
feat: add output format test help cmd

### LINKED ISSUE #586: Add supported output formats to cli help text
**Report type**
Missing documentation
 
**What can be improved?**  
When running `datree test -h` we get the following output:
```
Flags:
  -o, --output string                 Define output format
```

**Suggested solution**  
Add supported formats to help text (json, yaml, xml, JUnit)

### PR DESCRIPTION
**Pull request by amustaque97**
Fixes: #586

### CODE REVIEW

File: cmd/test/main.go [hunk 1]
```diff
@@ -222,9 +222,9 @@ func New(ctx *TestCommandContext) *cobra.Command {
 // AddFlags registers flags for a cli
 func (flags *TestCommandFlags) AddFlags(cmd *cobra.Command) {
 	if ciContext.Extract() != nil && ciContext.Extract().CIMetadata.ShouldHideEmojis {
-		cmd.Flags().StringVarP(&flags.Output, "output", "o", "simple", "Define output format")
+		cmd.Flags().StringVarP(&flags.Output, "output", "o", "simple", "Define output format (json, yaml, xml, JUnit)")
```
Comment (royhadad) [inline cmd/test/main.go]: some of the output options are missing (the issue description was wrong).
You should use similar code to the one in this line:
https://github.com/datreeio/datree/blob/4e0854a6cef47ef922713e5678d76c0e2c49a75a/cmd/test/main.go#L78

`strings.Join(evaluation.ExplicitOptionOptions, ", ")`

consider extracting it into a single function in `/bl/evaluation/output_flag.go`

**Review by royhadad — CHANGES_REQUESTED** [review]

**Comment by royhadad** [comment]
@amustaque97 are you still working on this PR or can we close it?

**Comment by amustaque97** [comment]
> @amustaque97 are you still working on this PR or can we close it?

Thanks for the ping. Looks like I lost the track. Let me continue working on the PR. I will make the required changes.

File: cmd/test/main.go (hunk 1 above)
Comment (amustaque97) [reply cmd/test/main.go]: I have addressed the comment. It is good for an another round of review.

**Review by royhadad — APPROVED** [review]

**Comment by royhadad** [comment]
thanks buddy 💪

**Pull request merged**